aries-site/src/examples/components/checkbox/CheckBoxToggleExample.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, FormField } from 'grommet';
+
+export const CheckBoxToggleExample = () => {
+  const [checked, setChecked] = useState(false);
+  return (
+    <Box align="center" pad="large">
```

Comment:
I don't think we need pad here.
```suggestion
     <Box width="medium" align="center">
```